src/m365/aad/commands/o365group/o365group-recyclebinitem-list.ts

```diff
@@ -0,0 +1,80 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { Group } from './Group';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  displayName?: string;
+  mailNickname?: string;
+}
+
+class AadO365GroupRecycleBinItemCommand extends GraphItemsListCommand<Group> {
+  public get name(): string {
+    return `${commands.O365GROUP_RECYCLEBINITEM_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Lists Microsoft 365 Groups deleted in the current tenant';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.displayName = typeof args.options.displayName !== 'undefined';
+    telemetryProps.mailNickname = typeof args.options.mailNickname !== 'undefined';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'displayName', 'mailNickname'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    let filter: string = `?$filter=groupTypes/any(c:c+eq+'Unified')`;
+    const displayNameFilter: string = args.options.displayName ? ` and startswith(DisplayName,'${encodeURIComponent(args.options.displayName).replace(/'/g, `''`)}')` : '';
+    const mailNicknameFilter: string = args.options.mailNickname ? ` and startswith(MailNickname,'${encodeURIComponent(args.options.mailNickname).replace(/'/g, `''`)}')` : '';
+    const topCount: string = '&$top=100';
+
+    let endpoint: string = `${this.resource}/v1.0/directory/deletedItems/Microsoft.Graph.Group${filter}${displayNameFilter}${mailNicknameFilter}${topCount}`;
+    
+
+    this
+      .getAllItems(endpoint, logger, true)
+      .then((): void => {
+        
+        logger.log(this.items);
+
+        if (this.verbose) {
+          logger.logToStderr(chalk.green('DONE'));
+        }
```

Comment:
Looks that these lines are not covered by the test coverage.

You can verify the coverage of the test by running all the tests using npm run test. I am getting this result
![image](https://user-images.githubusercontent.com/18241305/103330104-55b25b00-4a60-11eb-8538-17ffb5151654.png)